target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -45,6 +45,13 @@
 			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
 			pinctrl-names = "default";
 
```

Comment:
Maybe it's better to put it in both places? Since there are doubts which way is correct